package.json

```diff
@@ -96,6 +97,7 @@
     "reactjs-popup": "^2.0.5",
     "replay-next": "workspace:*",
     "reselect": "^4.1.5",
+    "semver": "^7.5.1",
```

Comment:
:hand: Can we use the much smaller https://www.npmjs.com/package/compare-versions lib I used over in the React DevTools?

Reply:
👍🏼  Sure.